src/Components/ArtworkAlert/Components/Modal.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  ModalBase,
+  ModalBaseProps,
+  splitBoxProps,
+  useDidMount,
+  Box,
+  DROP_SHADOW,
+  ModalClose,
+} from "@artsy/palette"
+import React from "react"
+
+export type ArtworkAlertModalProps = ModalBaseProps
+
+export const Modal: React.FC<ArtworkAlertModalProps> = ({
```

Comment:
**walkthrough**: Based heavily on [the similar component within the onboarding flow](https://github.com/artsy/force/blob/f6401bcbea2b364f88525e124c94ab50b1eb37b8/src/Components/Onboarding/Components/OnboardingModal.tsx).